app/src/main/java/nl/tudelft/b_b_w/controller/BlockController.java

```diff
@@ -8,11 +8,11 @@
 import nl.tudelft.b_b_w.model.Block;
 import nl.tudelft.b_b_w.model.BlockFactory;
 import nl.tudelft.b_b_w.model.DatabaseHandler;
+import nl.tudelft.b_b_w.model.TrustValues;
 import nl.tudelft.b_b_w.model.User;
 
 /**
  * Performs the actions of the blockchain
- * Created by jasper on 11/05/2017.
```

Comment:
Removing the copyright was a smart move. This way it seems the project is made entirely by our group.